eventstore/jpa/spring-data/src/main/java/org/occurrent/eventstore/jpa/springdata/StreamRepository.java

```diff
@@ -0,0 +1,17 @@
+package org.occurrent.eventstore.jpa.springdata;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@Repository
+public interface StreamRepository extends JpaRepository<StreamEntity, UUID> {
```

Comment:
I guess this class is internal and should not be used outside by users of the Occurrnt lib? If so, I think that either it should be moved to an "internal" package or remove "public" from the interface to make this more clear.